certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -930,93 +396,80 @@
+        "summary": "Get Organization relationships",
+        "description": "Get Organization Relationships",
+        "operationId": "Get-Organization-Relationships"
       }
     },
-    "/api/v1/Committees/{ID Or Code}/Members/{Page Number}": {
-      "get": {
+    "/api/v1/Organizations/{ID or Record Number}/Addresses": {
+      "post": {
         "tags": [
-          "Committees"
+          "Organizations"
         ],
         "responses": {
-          "default": {
+          "200": {
             "description": "default",
-            "schema": {}
+            "schema": {
+              "$ref": "#/definitions/AddressSaveData"
+            }
           }
         },
-        "summary": "Get Committee Members by committee ID or code",
-        "description": "Get Committee Members by Committee ID or Code",
-        "operationId": "Get-Committee-Members-by-committee-ID-or-code",
+        "summary": "Add or Update Address to Organization",
+        "description": "Add or Update Address to Organization",
+        "operationId": "Add-or-Update-Address-to-Organization",
         "parameters": [
           {
-            "name": "ID Or Code",
+            "name": "ID or Record Number",
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "Committee ID or Code",
-            "x-ms-summary": "Committee ID or Code",
-            "x-ms-url-encoding": "single"
+            "description": "Organization ID or Record Number.",
+            "x-ms-url-encoding": "single",
+            "x-ms-summary": "Organization ID or Record Number."
           },
           {
-            "name": "Page Number",
-            "in": "path",
+            "name": "Content-Type",
+            "in": "header",
             "required": true,
-            "type": "integer",
-            "default": 1,
-            "description": "Page Number",
-            "x-ms-summary": "Page Number",
-            "x-ms-visibility": "advanced",
-            "x-ms-url-encoding": "single"
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
           },
           {
-            "name": "Term",
-            "in": "query",
+            "name": "body",
+            "in": "body",
             "required": false,
-            "type": "integer",
-            "description": "Term",
-            "x-ms-summary": "Term"
-          },
-          {
-            "name": "Position Codes",
-            "in": "query",
-            "required": false,
-            "type": "string",
-            "description": "Position Codes",
-            "x-ms-summary": "Position Codes"
-          },
-          {
-            "name": "Content-Type",
-            "in": "header",
-            "required": true,
-            "type": "string",
-            "default": "application/json",
-            "x-ms-visibility": "internal"
+            "schema": {
+              "$ref": "#/definitions/AddressSaveData"
```

Comment:
I missed this one last time, can you add the enum options for the address type: Home, Work and Other.

Reply:
I can add the enum option to the ref "AddressSaveData", will this work? Or does this option not apply to all actions with "addresssavedata." If the latter is the case, I can remove the reference and add the schema with the enum in it.

Reply:
Yes, adding it to the ref would work